### DISCUSSION TITLE
Déduire la catégorie d'activité à partir du code NAF

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2620] [Idées]
Quand l'utilisateur renseigne son entreprise, on a l'information du code NAF fourni par l'API sirene. Il faudrait déduire automatiquement la catégorie d'activité à partir de ce dernier, tant que faire se peut.

- Pour la restauration / hotellerie, c'est direct : ils ont des codes dédié
- Pour les professions libérales, c'est plus délicat, mais c'est partiellement faisable : https://www.entreprises.gouv.fr/files/files/directions_services/cnapl/observatoire/etudes/observatoire-de-l-activite-liberale-guide-methodologique.pdf
- Pareil pour l'artisanat : https://www.entreprises.gouv.fr/files/files/directions_services/secteurs-professionnels/artisanat/nafa/nomenclature_nafa_rev2.pdf

L'idée est d'essayer d'inférer au maximum la classe, tout en demandant à l'utilisateur si on ne possède pas assez d'informations.

**Comment by mquandalle** [comment]
D'autant plus pertinent que l'on a maintenant des simulateurs dédiés pour les professionnels libéraux, que cette fonctionnalité permettrait de référencer automatiquement après la saisie du nom de la personne dans la section « Gérer »

**Comment by wiinxt** [comment]
Je sais pas si c'est d'actualité mais j'ai les codes NAF, CPF et APE dans les données du POC de la recherche du code APE https://github.com/betagouv/mon-entreprise/pull/2380
